ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -10754,6 +10754,7 @@ periodics:
     org: openshift
     repo: release
   labels:
+    dummy: "true"
```

Comment:
That didn't cause pj-rehearse to pick the job that we want to rehearse, unfortunately. Let me think of what we can do to verify locally that this change will work.

Reply:
Let's remove this, since it didn't help with rehearsals.